tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,25 +87,26 @@ func (tt *tester) runLoop() {
 			for _, s := range tt.cluster.Stressers {
 				s.Cancel()
 			}
+			log.Printf("etcd-tester: [round#%d case#%d] canceled stressers", i, j)
 
+			log.Printf("etcd-tester: [round#%d case#%d] checking current revisions...", i, j)
 			ok := false
+			var currentRevision int64
 			for k := 0; k < 5; k++ {
 				time.Sleep(time.Second)
-				log.Printf("etcd-tester: [round#%d case#%d.%d] checking current revisions...", i, j, k)
 				revs, err := tt.cluster.getRevision()
 				if err != nil {
 					if e := tt.cleanup(i, j); e != nil {
 						log.Printf("etcd-tester: [round#%d case#%d.%d] cleanup error: %v", i, j, k, e)
 						return
 					}
-					log.Printf("etcd-tester: [round#%d case#%d.%d] failed to get revisions (%v)", i, j, k, err)
+					log.Printf("etcd-tester: [round#%d case#%d.%d] failed to get current revisions (%v)", i, j, k, err)
 					continue
 				}
-				if ok = isSameValueInMap(revs); ok {
-					log.Printf("etcd-tester: [round#%d case#%d.%d] checking current revisions succeed!", i, j, k)
+				if currentRevision, ok = isSameValueInMap(revs); ok {
```

Comment:
let's change the func name. IsXX should only return a bool. Change this to getSameValue.